README.md

```diff
@@ -5,6 +5,16 @@ author: dvana
 ---
 #Get started with Azure Data Catalog
 
-This sample shows you how to Register, Search, and Delete a data asset using the Data Catalog REST API.
+This sample shows you how to use the Azure Data Catalog REST API to Register, Search, and Delete data assets.
+
+To get started using this sample, you first need to build the executable, following below steps:
+
+* Download the sample project
+* Open it in Visual Studio and install the latest version of dependent NuGet packages (e.g Microsoft.IdentityModel.Clients.ActiveDirectory) 
```

Comment:
Resolved #1 and #2 
#3 is By Design and the sample code is well annotated describing how to install the dependent NuGet packages.
 (e.g //  run "Install-Package Microsoft.IdentityModel.Clients.ActiveDirectory" from the NuGet Package Manager Console.)

Reply:
Let's compare.

Path 1 - The dev is warned right here and provided in the readme with the appropriate command to fix the NuGet issue before they compile and encounter any errors

Path 2 - The dev tries to build the project. Fails because of some dependency issue. And is then expected to go find the right file which happens to contain the correct command to figure out where the notice about the fix is.

I am struggling to understand how path 2 can be considered superior to path 1. Why would we intentionally put blocks in the way of the dev getting quickly started?